packages/datepicker/src/react/calendar.tsx

```diff
@@ -46,43 +42,28 @@ const styles = {
 }
 
 interface CalendarProps {
-  date: DateParts
-  onSelect: (value: string | undefined) => void
+  value: Date | undefined
```

Comment:
This can be optional as well `value?: Date`

Reply:
Ditto.